seshdash/urls.py

```diff
@@ -23,5 +23,7 @@
             url(r'^get-latest-bom-data', views.get_latest_bom_data),
             url(r'^search', views.search),
             url(r'^historical_data', views.historical_data),
+            url(r'^influx$', views.influx),
```

Comment:
Please name these different. Something like ``get_measurments` and get_measurment_values